Cargo.lock

```diff
@@ -129,6 +129,28 @@ dependencies = [
  "byteorder 1.3.1 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "bindgen"
```

Comment:
New dependency, which I'm 👍  to adding but which we should talk about due-dilligence for explicitly in the PR.